automatic/graphviz/tools/chocolateyInstall.ps1

```diff
@@ -16,7 +16,10 @@ Remove-Item $toolsPath\*.exe -ea 0
 
 $packageName = $packageArgs.packageName
 $installLocation = Get-AppInstallLocation $packageArgs.softwareName
-if (!$installLocation)  { Write-Warning "Can't find $packageName install location"; return }
+if (!$installLocation)  { Write-Warning "Can't find $packageName install location"; return 1 }
```

Comment:
With this change you may have broken the package.
The package may fail to install when you return `1` in this context, even though the software have already been installed.
Please revert this change.

Reply:
Fixed. Thank you for the feedback!